Sources/TokamakDOM/DOMRenderer.swift

```diff
@@ -59,16 +59,21 @@ public final class DOMRenderer: Renderer {
 
   public init<V: View>(_ view: V, _ ref: JSObjectRef) {
     rootRef = ref
-    rootRef.style = "display: flex; width: 100%; height: 100%; justify-content: center; align-items: center; overflow: hidden;"
+    rootRef.style = "display: flex; width: 100%; height: 100%;"
+      + "justify-content: center; align-items: center; overflow: hidden;"
```

Comment:
Multiline strings are easier to read, and this also would fix the compilation error 🙂

```suggestion
    rootRef.style = """
    display: flex; width: 100%; height: 100%;
    justify-content: center; align-items: center; overflow: hidden;
    """
```